misc/config

```diff
@@ -0,0 +1,14 @@
+# for fast testing, you should disable these options for production environments
```

Comment:
What's this file for?

Reply:
It is the `.ssh/config` sample file from documentation of the module inside this section :

```
10. Fill the `ssh/config` file from `mibuploader` directory and configure SSH commands into configuration. The SSH keys copy is mandatory.
```